### PR TITLE
DIV-6857 Update to 'Record Judge cost order decisions'

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6857

### Change description ###

**Update:** Due to CCD limitations, the questions need to be moved on different pages. 

Current logic: 
**Page 1:**
- GrantCostOrder question

**Page 2:**
-> show current page on if GrantCostOrder == Yes

- if GrantCostOrder=Yes and is an adultery case 
-> show WhoShouldPay question

 - show judgeCostOrderDecision question 
Judge should be able to make a cost oder decision (_judgeCostOrderDecision_ field) ONLY if Cost Order is Granted
Additional Info should only appear if  _Additional information / free text order_ is selected 

Screenshot 2021-04-01 at 15 18 16

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-object-to-costs-nonprod.json
Comment (ravinderpal) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-object-to-costs-nonprod.json:47]: Seems like parenthesis are not supported so I had to do `A && C || B && C ` instead of `( A || B ) && C `

Comment (ravinderpal) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields-object-to-costs-nonprod.json]: Can't use operator AND and OR together due to CCD limitation hence moving questions to different pages (same as `LA Make a Decision`)

**Comment by ravinderpal** [comment]
Ticket on hold.

**Review by madjava — APPROVED** [review]

**Review by yarpo — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

File: package.json
Comment (ravinderpal) [inline package.json:22]: For testing only. To be removed before merging

**Review by yarpo — APPROVED** [review]

**Pull request merged**
